runtime/include/http_session.h

```diff
@@ -456,26 +418,10 @@ static inline int
 http_session_write_response(struct http_session *session, const uint8_t *source, size_t n)
```

Comment:
This func became a zombie function now, since the only occurrence in "wasi_impl_serverless.c:813" was removed.

Reply:
Reverted back the wasi_impl code, since the function has some assertions.